packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -101,7 +102,7 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                     id='dt_start_time_label'
                     icon={<Icon icon='IcContractStartTime' size={24} />}
                     label={localize('Start time')}
-                    value={toGMTFormat(epochToMoment(purchase_time)) || ' - '}
+                    value={toGMTFormat(epochToMoment(date_start || purchase_time)) || ' - '}
```

Comment:
How come we need this? We shouldn't be allowing users to view Contract Details for unsupported contracts, in which forward starting contracts is one of them.

Reply:
Should we disallow the client to view the contract after the contract ended? I prevented the client to view the contract detail from the `open positions` tab. Should I do the same for the `profit table` tab (the details for the ended contracts are correct.)?

Reply:
For reports, when it comes to open or expired contracts / positions, they shouldn't be able to access Contract Details if they click on the table row, it should toggle a message instead in the table-row content.

For Recent positions, the correct behaviour is toggling the `UnsupportedContractModal` that has the CTA options for them to stay or view the contract in Binary.com.

So we don't actually need the `date_start` value here, since clients shouldn't be able to reach the Contract Details when it comes to unsupported contracts.

Reply:
@ali-fs , you can refer to `contract.js` [here](https://github.com/binary-com/deriv-app/blob/master/packages/trader/src/Constants/contract.js), we just need to add an additional helper function to check if the response from `proposal_open_contract` contains `is_forward_starting`. 

If it's there, we can use the same `UnsupportedContractModal` [here](https://github.com/binary-com/deriv-app/blob/master/packages/trader/src/App/Components/Elements/Modals/UnsupportedContractModal/unsupported-contract-modal.jsx) to prevent users from viewing the contract. 

The reason we want to do this is because we aren't handling forward starting contracts in the chart markers and contract details, so it's better to prevent users from seeing any discrepancies with their contracts while it's unsupported.